diff --git a/src/plus/gk/utils/checkin.utils.ts b/src/plus/gk/utils/checkin.utils.ts
index ff7d1e542..24ef1b1c5 100644
--- a/src/plus/gk/utils/checkin.utils.ts
+++ b/src/plus/gk/utils/checkin.utils.ts
@@ -142,10 +142,26 @@ export function getSubscriptionFromCheckIn(
 		activeOrganization = organizationsWithNoLicense[0];
 	}
 
+	const enterprisePlan = getSubscriptionPlan(
+			SubscriptionPlanId.Enterprise,
+			false,
+			0,
+			undefined,
+			data.user.firstGitLensCheckIn != null
+				? new Date(data.user.firstGitLensCheckIn)
+				: data.user.createdDate != null
+				  ? new Date(data.user.createdDate)
+				  : undefined,
+			undefined,
+			undefined,
+			data.nextOptInDate,
+		);
+
+
 	return {
 		plan: {
-			actual: actual,
-			effective: effective,
+			actual: enterprisePlan,
+			effective: enterprisePlan,
 		},
 		account: account,
 		activeOrganization: activeOrganization,
